doni/objects/hardware.py

```diff
@@ -60,7 +62,8 @@ def save(self, context: "RequestContext" = None):
             HardwareDuplicateName: if a hardware with the same name exists.
             HardwareNotFound: if the hardware does not exist.
         """
-        updates = self.obj_get_changes()
+        updates = self.obj_get_changes() # BUG: Does not detect changes to hardware properties
+        updates["properties"] = self.properties
```

Comment:
Can you extract this to its own commit with more info?

Reply:
Done!